docs/Windows.md

```diff
@@ -1,43 +1,88 @@
+
+#### Downloading the nightlies
 
-Alternatively, to make these changes persistent go to the Windows environment variables edit dialog found in `System Properties` > `Advanced` > `Environment Variables` and edit the path variable to add the following two new items.
+1. Go to [https://dev.azure.com/compnerd/windows-swift](https://dev.azure.com/compnerd/windows-swift).
+2. Choose `Pipelines` > `Pipelines` from the left of the dashboard.
+3. Choose 'Runs'.
+4. Use the filter symbol to search for "VS2017".
+5. In the results list, scroll down until you see the first successful build (with a green OK symbol) that does not have "facebook" in the description and click on it.
+6. Under "Artifacts", click on the "... published" link.
+7. Download windows-toolchain-amd64.msi, windows-sdk.msi, and windows-runtime-amd64.msi by clicking on the appropriate down-arrows on the right. Be sure to really download these files from the same build (i.e. do not switch the build for the next download, and be careful when updating). These files will be downloaded as zip files. Unless they are not automatically unzipped during the download process, unzip them to obtain \*.msi files in the extracted directories.
 
-```cmd
-C:\Library\Developer\Toolchains\unknown-Asserts-development.xctoolchain\usr\bin
-C:\Library\icu-64.2\bin
-```
+#### Installing the nightlies
+
+These \*.msi files install the files to `C:\Library`. The complete Library directory can later be copied to a different location to be used there (but you have to change the PATH environment variable then).
+
+To install the Swift Toolchain and Swift Windows SDK, run the installers windows-toolchain-amd64.msi and windows-sdk.msi. To install the Swift Runtime, run the installer windows-runtime-amd64.msi.
+
+The installation of the Swift Toolchain adds the following value to the system PATH environment variable:
+
+`\Library\Developer\Toolchains\unknown-Asserts-development.xctoolchain\usr\bin`
+
+The installation of the Swift Runtime adds the following value to the system PATH environment variable:
+
+`\Library\Swift\Current\bin`
+
+#### Further requirements: CMake
+
+On Windows, the most convenient setup for building Swift projects currently involves the use of CMake. This requires CMake 3.15+ for Swift support. You can download CMake from <https://cmake.org/>.
+
+#### Further requirements: ICU
+
+You will need the ICU libraries from <http://site.icu-project.org/>. The nightlies are built against ICU 64.2 from the ICU project. You can download the binaries for that from http://download./files/icu4c/64.2/icu4c-64_2-Win64-MSVC2017.zip. In these instructions we assume you rename the extracted ICU folder `icu4c-64_2-Win64-MSVC2017` to `icu-64.2` and move it to `C:\Library` and that you rename `bin64` to `bin`. The path to that `bin` should be added to the PATH environment variable.
```

Comment:
Please use the markdown syntax for the ICU project's website.  You should fix the URL for the download.  Additionally, perhaps that should be a link?